docs/device-agent/quickstart.md

```diff
@@ -8,60 +8,100 @@ meta:
       - flowfuse
 ---
 
-# Developing Flows on Your Device
+# Quick Start Guide: Device Agent
 
-## Add Device to FlowFuse
+This guide will walk you through the process of adding a device to FlowFuse, connecting it to the platform, and deploying your Node-RED flows remotely. FlowFuse's Device Agent helps unlock the power of your devices by allowing you to manage and deploy Node-RED flows running on those devices securely and remotely.
 
-### Prerequisites
+![FlowFuse Device Agent Workflow](./images/device-agent-workflow.png){data-zoomable}
+_Workflow to onboard new Edge Devices through the FlowFuse Platform and Device Agent_
 
-If you do not already have the device agent installed, follow the [install instructions](./install.md), otherwise, proceed to the next step.
+## Setup & Installation
 
-_NOTE: It is good practice to keep your Device Agent up to date. Update instructions are also provided in the link above._
+### Step 1: Install Device Agent (On Device)
 
-_NOTE: If you prefer to use the Device Agent WEB UI, follow the [Quick Start with Web UI](./quickstart-web-ui.md) guide._
+NodeJS of version 20 or later is recommended, through the device agent supports NodeJS v14 or later.
+
+#### Linux/MacOS
+
+```bash
+sudo npm install -g @flowfuse/device-agent
+```
+
+#### Windows
```

Comment:
The above linux instruction includes `sudo` which infers administrator access however the windows version does not make this clear.

I appreciate the full install instructions are linked below however a nod to administrative command prompt would be a helpful hint to avoid 1st try failures.

```suggestion
#### Windows (Elevated command prompt)
```

Reply:
Rather than putting this in the header, I'll put it in as a line of text

Reply:
```suggestion
#### Windows

Issue the below command in an elevated command prompt:
```

Does this help?